WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -16045,14 +16069,6 @@ e_armenia = {
 		zoroastrian_group = 200 # Crusade target weight
 		
 		# Creation/usurpation trigger
-		allow = {
-			hidden_tooltip = {
-				OR = {
-					ai = no
-					religion_group = christian
-				}
-			}
-		}
```

Comment:
Here is what it shows in my code:

	k_caucasian_albania = {
		color={ 172 82 55 }
		color2={ 255 255 255 }
		
		capital = 668 # Shirvan
		
		culture = armenian
		
		orthodox = 300 # Crusade target weight
		muslim = 100 # Crusade target weight
		zoroastrian_group = 200 # Crusade target weight
		
		# Creation/usurpation trigger
		
		d_caucasian_albania = {
			color={ 195 125 55 }
			color2={ 255 255 255 }
			
			capital = 668 # Shirvan

			c_shemakha = {
				color={ 168 240 28 }
				color2={ 255 255 255 }